src/extensions/anvil/prop.ts

```diff
@@ -67,13 +67,31 @@ export const MembraneOrientationProvider: CustomStructureProperty.Provider<Membr
     type: 'root',
     defaultParams: MembraneOrientationParams,
     getParams: (data: Structure) => MembraneOrientationParams,
-    isApplicable: (data: Structure) => true,
+    isApplicable,
     obtain: async (ctx: CustomProperty.Context, data: Structure, props: Partial<MembraneOrientationProps>) => {
         const p = { ...PD.getDefaultValues(MembraneOrientationParams), ...props };
-        return { value: await computeAnvil(ctx, data, p) };
+        try {
+            return { value: await computeAnvil(ctx, data, p) };
+        } catch (e) {
+            // the "Residues Embedded in Membrane" symbol may bypass isApplicable() checks
+            console.warn('Failed to predict membrane orientation. This happens for short peptides and entries without amino acids.');
+            return { value: undefined };
+        }
     }
 });
 
+function isApplicable(structure: Structure) {
+    if (!structure.isAtomic) return false;
+
+    const { byEntityKey } = structure.model.sequence;
```

Comment:
Could you iterate over all models? Also, accessing .model when there is more than one model will throw (@dsehnal how about we change it so that it return undefined?)

Reply:
Thanks, updated with https://github.com/molstar/molstar/pull/1328/commits/2ed165f9a5ff78815e6246cdf7308127136f140e.